default/techs.txt

```diff
@@ -3152,6 +3152,31 @@ Tech
             effects = SetMaxCapacity partname = "SR_WEAPON_4_1" value = Value + 5
     graphic = "icons/ship_parts/death-ray-4.png"
 
+Tech
+    name = "SHP_ORGANIC_WAR_ADAPTION"
+    description = "SHP_ORGANIC_WAR_ADAPTION_DESC"
+    short_description = "SHP_ORGANIC_WAR_ADAPTION_SHORT_DESC"
+    techtype = Application
+    category = "SHIPS_CATEGORY"
+    researchcost = 75 * [[TECH_COST_MULTIPLIER]]
+    researchturns = 5
+    prerequisites = [
+        "SHP_WEAPON_2_2"
+        "SHP_ORG_HULL"
+    ]
+    unlock = Item type = ShipPart name = "SP_SOLAR_CONCENTRATOR"
```

Comment:
In one case you use underscore after SOLAR in other not.  IMHO it will be better to make it consistent.
```SP_SOLAR_CONCENTRATOR
SHP_SOLARCONNECTION```